bitwarden-server.sln

```diff
@@ -13,10 +13,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
 		.gitignore = .gitignore
-		build.sh = build.sh
 		.github\workflows\build.yml = .github\workflows\build.yml
 		Directory.Build.props = Directory.Build.props
-		dotnet-tools.json = dotnet-tools.json
 		LICENSE.txt = LICENSE.txt
```

Comment:
Why were these lines removed?

Reply:
@MGibson1, This section represents "Solution items". I noticed that some files no longer exist
![image](https://user-images.githubusercontent.com/25577658/217269326-234e8a75-abde-40f2-9a8f-c40f90023d7b.png)

Reply:
Hmm. It seems these files have been moved over the years.

Each project has a build.sh and dotnet-tools.json has been moved to `.config/dotnet-tools.json`

However, like the other change, I'd prefer to keep the dependency update atomic, can you revert this and we can figure out what to do about this in a separate PR?